### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Tanmay09516**
## Fixes Issue
completes a part of #236 

## Changes proposed
In readme of backend development
- added what I mentioned about describing the resources already provided
- also corrected the spelling of resources (it was resourse before)

## Note to reviewers

I have not completed the entire readmes I have only completed one as of now, I thought getting it checked would be better so I could get any changes the reviewers would like in this and the other readmes.

### CODE REVIEW

File: Backend-Development/README.md [hunk 1]
```diff
@@ -12,25 +12,25 @@
 - [HTML](https://www.youtube.com/watch?v=kUMe1FH4CHE)
 
 ### Build a project
-- [Resourse 1](https://www.frontendpractice.com/)
-- [Resourse 2](https://www.frontendmentor.io/)
+- [Resource 1 : frontendpractice.com](https://www.frontendpractice.com/)
```
Comment (siddhant-khisty) [inline Backend-Development/README.md:15]: Remove the `Resource 1`, `Resource 2` tags, and just add the resource name.
For example:
Before: `Resource 1: Javascript Course `
After: `Javascript Course`

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
Suggested some changes. Please apply these changes to the entire file

File: Backend-Development/README.md (hunk 1 above)
Comment (Tanmay09516) [reply Backend-Development/README.md:15]: on it sir!

**Review by siddhant-khisty — APPROVED** [review]
LGTM
cc @kaiwalyakoparkar

**Review by J0SAL — APPROVED** [review]

**Comment by Tanmay09516** [comment]
I have completed the task from my end please let me know if any more changes are needed.

**Pull request merged**